provisioning_templates/provision/autoyast_sles_default.erb

```diff
@@ -186,6 +186,9 @@ cp /etc/resolv.conf /mnt/etc
 
 <%= snippet('remote_execution_ssh_keys') %>
 
+#blacklist kernel modules
+<%= snippet "blacklist_kernel_modules" %>
```

Comment:
No SUSE expert here, but isn't autoyast a XML syntax? This comment looks like kickstart one.

Reply:
You are right. AutoYAST ist XML syntax but this change is in a script section which are just common shell commands.